feconf.py

```diff
@@ -72,7 +72,10 @@
 
 # A mapping of interaction ids to their default classifier.
 INTERACTION_CLASSIFIER_MAPPING = {
-    'TextInput': 'LDAStringClassifier'
+    'TextInput': {
+        'classifier_id' : 'LDAStringClassifier',
```

Comment:
classifier_id can be confused with the id of instance of classifier model. algorithm_id might be a better term.